tools/runtime_configs/python/3.8.10

```diff
@@ -0,0 +1,17 @@
+export PYTHON_CONFIGURE_OPTS="\
```

Comment:
This file is from the older CodeBuild Ubuntu Standard 2.0 image:
https://github.com/aws/aws-codebuild-docker-images/blob/9282872af78aeb1b5df3010ed3872c40f3d0f056/al2/x86_64/standard/2.0/tools/runtime_configs/python/3.8.10